Build/BUILDmom6

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+#***********************************************************************
+#*                   GNU Lesser General Public License
+#*
+#* This file is part of the SHiELD Build System.
+#*
+#* The SHiELD Build System free software: you can redistribute it
+#* and/or modify it under the terms of the
+#* GNU Lesser General Public License as published by the
+#* Free Software Foundation, either version 3 of the License, or
+#* (at your option) any later version.
+#*
+#* The SHiELD Build System distributed in the hope that it will be
+#* useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+#* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#* See the GNU General Public License for more details.
+#*
+#* You should have received a copy of the GNU Lesser General Public
+#* License along with theSHiELD Build System
+#* If not, see <http://www.gnu.org/licenses/>.
+#***********************************************************************
+#
+#  DISCLAIMER: This script is provided as-is and as such is unsupported.
+#
+set -x
+#
+# set default values
+# configure your build parameters
+  COMPILER="intel"
+  BIT="64bit"
+#
+# parse arguments
+  for arg in "$@"
+  do
+      case $arg in
+          intel|gnu|nvhpc)
```

Comment:
I think you should remove nvhpc as a compile option for the SHiELD-mom6 configuration.  And if you do remove it I think you should have the compile script exit if someone tries to compile shield-mom6 with nvhpc.  You can do this similarly to how we exit if someone tries to compile with shield and sw options: 
https://github.com/NOAA-GFDL/SHiELD_build/blob/41662e11510c5c808eccf132a2ba9d167bdbfaed/Build/COMPILE#L138-L141
 @bensonr do you agree?

Reply:
done